src/test/java/org/broadinstitute/hellbender/tools/exome/ExomeToolsTestUtils.java

```diff
@@ -34,42 +34,57 @@ protected static File getTestDataDir(){
      */
     protected final static SAMSequenceDictionary REFERENCE_DICTIONARY = SAMSequenceDictionaryExtractor.extractDictionary(REFERENCE_FILE);
 
+    protected static Comparator<SimpleInterval> createIntervalComparator() {
```

Comment:
This is the second time a comparator like this shows up.  I think it should probably just be extracted out of here and put into `SimpleInterval`.

I think it can be more clearly expressed as 

```
 Comparator<SimpleInterval> comparator = Comparator.comparing(SimpleInterval::getContig)
.thenComparingInt(SimpleInterval::getStart)
.thenComparingInt(SimpleInterval::getEnd);
```